x/nameservice/handler.go

```diff
@@ -55,7 +56,11 @@ func handleMsgBuyName(ctx sdk.Context, keeper Keeper, msg MsgBuyName) sdk.Result
 
 // Handle a message to delete name
 func handleMsgDeleteName(ctx sdk.Context, keeper Keeper, msg MsgDeleteName) sdk.Result {
-    // Checks if the the msg sender is the same as the current owner
+	// Checks if the name is present in the store or not
+	if !keeper.IsNamePresent(ctx, msg.Name) {
+		return types.ErrNameDoesNotExist(types.DefaultCodespace).Result()
+	}
+    // Checks if the msg sender is the same as the current owner
```

Comment:
ditto

Reply:
Cool, will remove the unnecessary comments.